test/mlj/regression.jl

```diff
@@ -35,6 +35,9 @@ rmse_weights             = calc_rmse(y[test], yhat_with_weights)
 @test rmse < 0.05
 @test !isapprox(rmse, rmse_weights, rtol=0.1) # check that they differ by at least around 10% with/without weights
 
+# checking that fitting with and without weight outputs are non-identical
+@test yhat_with_weights != yhat
```

Comment:
The test has been done differently by showing that the RMSE varies substantially, so we dont need this additional test

Reply:
( see two lines above)

Reply:
Yes, I saw the test was done differently on this one. I added this part of troubleshooting the issue with the binary test. To make sure it wasn't a problem with multiclass, or regression. Makes sense to remove it

Reply:
16ddb07 resolves